tests/system/action/assignment/test_delete.py

```diff
@@ -3,25 +3,32 @@
 
 class AssignmentDeleteActionTest(BaseActionTestCase):
     def test_delete_correct(self) -> None:
-        self.create_model("assignment/111", {"title": "title_srtgb123"})
+        self.create_model("meeting/110", {})
+        self.create_model(
+            "assignment/111", {"meeting_id": 110, "title": "title_srtgb123"}
+        )
```

Comment:
Maybe it looks better to use set_models if there are more than one model actions

Reply:
Done, use now set_models in test_delete.